docs/amber_script.md

```diff
@@ -98,8 +98,6 @@ either image buffers or, what the target API would refer to as a buffer.
  * vec[2,3,4]\<type>
  * mat[2,3,4]x[2,3,4]\<type>    -- useful?
 
-TODO(dneto): Support half-precision floating point.
```

Comment:
An upcoming CL allows this to be a image format as well. So this could be done with R16_SFLOAT as the format.